### PR TITLE
Yy feature login

### PR DESCRIPTION
**Pull request by verasunyy**
add sign-in and sign-up screen,
add authentication in the root nav,
add asyncStorage for user token,
add Loader Component,
fix keyboard cover content issue,
add style to sign-in and sign-up page,
remove all console log and commented code and change the file name to camalCase

### CODE REVIEW

File: client/js/App.js [hunk 1]
```diff
@@ -1,8 +1,20 @@
...
+import SignIn from './screens/SignIn';
+import SignUp from './screens/SignUp';
 import RootStackNavigator from '../js/navigation/RootStackNavigator';
 
 const App = () => {
-  return <RootStackNavigator />;
+  return (
+    <ApolloProvider client={client}>
+      {/* <SignIn /> */}
+      {/* <SignUp /> */}
```
Comment (siddharthparmar7) [inline client/js/App.js]: remove this commented code.

File: client/js/App.js [hunk 2]
```diff
@@ -1,8 +1,20 @@
-import React, {Fragment} from 'react';
+
+import React, { Fragment } from 'react';
+
+import { ApolloProvider } from '@apollo/react-hooks'
+import client from './config/api'
+import SignIn from './screens/SignIn';
+import SignUp from './screens/SignUp';
```
Comment (siddharthparmar7) [inline client/js/App.js]: remove unnecessary imports.

File: client/js/config/models/Authentication.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import AsyncStorage from '@react-native-community/async-storage';
+
+export const createToken = async (token) => {
+  try {
+    await AsyncStorage.setItem('userToken', JSON.stringify({ id: token }))
+    console.log(token)
```
Comment (siddharthparmar7) [inline client/js/config/models/Authentication.js]: remove this console log.

File: client/js/config/models/Authentication.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import AsyncStorage from '@react-native-community/async-storage';
```
Comment (siddharthparmar7) [inline client/js/config/models/Authentication.js]: use camelCase for your file names in models dir.

File: client/js/screens/AuthLoading/AuthLoading.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  }
+
+  navigateUser = async () => {
+    const userToken = await AsyncStorage.getItem('userToken');
+    this.props.navigation.navigate(userToken ? 'App' : 'Auth');
+  };
+
+  render() {
+    return (
+      <View>
```
Comment (siddharthparmar7) [inline client/js/screens/AuthLoading/AuthLoading.js]: remove View component wrapper. Create a Loader component if needed.

File: client/js/screens/Profile/Profile.js [hunk 6]
```diff
@@ -1,12 +1,24 @@
...
+import { withNavigation } from 'react-navigation';
+const Profile = ({ navigation }) => {
   return (
     <View>
+      <Button
+        title={"Log Out"}
+        onPress={
+          async () => {
+            await deleteToken();
+            // const userToken = await queryToken()
```
Comment (siddharthparmar7) [inline client/js/screens/Profile/Profile.js]: remove commented code.

File: client/js/screens/SignIn/SignIn.js [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+    super(props);
+    this.state = {
+      error: null
+    };
+  }
+
+
+  render() {
+    const { navigation } = this.props;
+    console.log(navigation)
```
Comment (siddharthparmar7) [inline client/js/screens/SignIn/SignIn.js]: remove console log.

File: client/js/screens/SignIn/SignIn.js [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+                // if (!values.confirm) {
+                //   errors.confirm = 'Required'
+                // } else if (values.confirm !== values.password) {
+                //   errors.confirm = 'Must match'
+                // }
+                return errors
+              }}
+              onSubmit={async (values) => {
+                console.log("submit")
+                console.log("submit", values.email)
```
Comment (siddharthparmar7) [inline client/js/screens/SignIn/SignIn.js]: remove console logs.

File: client/js/screens/SignIn/SignIn.js [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+                  <TouchableHighlight
+                    onPress={handleSubmit}
+                  >
+                    <Text>Sign In</Text>
+                  </TouchableHighlight>
+
+
+                  <TouchableHighlight
+                    onPress={() => {
+                      console.log("jsbdfjhdakjfhd")
```
Comment (siddharthparmar7) [inline client/js/screens/SignIn/SignIn.js]: remove console log.

File: client/js/screens/SignIn/SignInContainer.js [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+
+class SignInContainer extends Component {
+  render() {
+    return (
+      <SignIn />
+    )
+  }
+}
+
+export default SignInContainer;
```
Comment (siddharthparmar7) [inline client/js/screens/SignIn/SignInContainer.js:11]: remove all the commented code.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Nice work! Please make the changes. Thank you.

File: client/js/App.js (hunk 1 above)
Comment (verasunyy) [reply client/js/App.js]: done

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**